components/modal/modal.js

```diff
@@ -184,6 +186,7 @@ Modal.defaultProps = {
   closeOnOverlayClick: true,
   footer: null,
   fullscreen: false,
+  mods: [],
```

Comment:
Well in this case it wouldn't happen, because of this: https://github.com/Travix-International/travix-ui-kit/pull/63/files#diff-aeb4723b4b18ec633e3c8bab2ea276e6R145

Nevertheless, to maintain consistency and prevent future problems, I agree with you @asci